source/taskmanager/pom.xml

```diff
@@ -30,6 +30,10 @@
   <name>Taskmanager :: Application</name>
   <description>AERIUS Taskmanager actual manager/scheduler (standalone)</description>
 
+  <properties>
+    <java.version>17</java.version>
```

Comment:
Do you only want the taskmanager set to 17.. and keep the client on 11?

I always thought:
```
    <maven.compiler.source>17</maven.compiler.source>
    <maven.compiler.target>17</maven.compiler.target>
```
was the way to go.. is this something implicit as well? I can't find any references to `java.version`.

Reply:
Ahh, nvm.. root pom :)

Reply:
I kept the client to 11 because of dependencies in calculator (calculator-wui-server). Although maybe better to use default 17 and set client specifically to 11.